src/lib/messages/jaia_dccl.proto

```diff
@@ -13,7 +13,7 @@ message Command
 {
     option (dccl.msg) = {
         id: 80
-        max_bytes: 200
+        max_bytes: 552
```

Comment:
We can't arbitrarily make this larger than the MTU supported by the XBee driver.

This needs to stay 200, and max_repeat for goal 14.

You can set more than 14 goals on hub when we're encoding using Protobufs.